platform/server/http/HttpServer.ts

```diff
@@ -63,8 +61,17 @@ export class HttpServer {
   }
 
   private initializeServer(config: HttpConfig) {
+    // TODO This was forced to `any` because of the following problem with the
```

Comment:
If I've got some time later this week I'll see if I can fix this problem in the expressjs declarations. I just wanted to get TS 2.6.1 in.

Reply:
With such a good clarifying comment, it's good for now! 👍

Reply:
Can you comment on where the express declarations are?

Reply:
It's the `@types/express` package. Source code here: https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/express.